whitepapers/Tables.md

```diff
@@ -1,5 +1,114 @@
 # SQL Tables
 
+## Template
 
+### {ModelName}
+{Description}
 
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| | | ✓ | ✓ | ✓ | | |
 
+#### Other indices
+* `{column_name}`, `{column_name}`, ... [(unique)]
+* ...
+
+## Data
+
+### SourceUniqueIdentifier (SUID)
+Identifier for a specific document from a specific source.
+
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| `source_doc_id` | text |  |  |  |  | Identifier given to the document by the source |
+| `ingest_config_id` | int |  |  | ✓ |  | IngestConfig used to ingest the document |
```

Comment:
That makes sense... just need to make sure the Regulator is in on it and preserves empty m2ms.

It could still be an issue if (stretching my example even deeper into unlikelihood) A has simple affiliation data (to a university, say), while B has affiliations to university, department, school, and lab. A's short list would overwrite B's long one.